src/components/wallet/wallet.js

```diff
@@ -223,24 +265,43 @@ export default class Wallet extends Component {
             <div
                 className="wallet"
             >
-                <button
-                    onClick={ () => wallet.isSignedIn()
-                        ? this.signOut() 
-                        : this.signIn()
-                    }
-                >
-                    { window.NEAR_ENV === "testnet" 
-                        ? <ScienceOutlinedIcon/>
-                        : <AccountBalanceWalletOutlinedIcon/>
-                    }
-                    { wallet.isSignedIn()
-                        ? wallet.getAccountId()
-                        : `Sign in`
-                    }
-                </button>
-                { wallet.isSignedIn()
-                    ? <>
-                        <span className="for">for</span>
+                <div className="user" expand={ expanded.user || !wallet.isSignedIn() ? "yes" : "no" }>
```

Comment:
When switching App <-> DAO page, both user & DAO selectors expand automatically. This shouldn't happen

Reply:
Also I think it's better to hide addresses when it's not expanded

Reply:
> When switching App <-> DAO page, both user & DAO selectors expand automatically. This shouldn't happen
@yukamend could you please take care of this? This happens because the because the sidebars on the DAO and APP page are not the same element (see `src/pages`), u can also remove the `full={true}` and corresponding code in the sidebar component as it is no longer needed.

Reply:
Am I missing something or is this easily fixed by setting 'expanded.user' and 'expanded.dao' to false by default?

Reply:
@yukamend Maybe, but because of the unusual placement (people are used to login from the header), I think it's important to have them expanded per default if they don't have a value. So if user isn't logged in, expand per default (and same goes for DAO)